packages/polyfills/package.json

```diff
@@ -4,14 +4,7 @@
   "description": "Verified polyfills for the Element Framework",
   "author": "Refinitiv",
   "license": "Apache-2.0",
-  "main": "entrypoints/all.js",
-  "exports": {
-    ".": {
-      "default": "./entrypoints/all.js"
-    },
-    "./all.js": "./entrypoints/all.js",
-    "./minimal.js": "./entrypoints/minimal.js"
-  },
+  "main": "all.js",
```

Comment:
@Theeraphat-Sorasetsakul What do you think if we change `all.js` into `index.js`. To me, it looks cleaner that way.

Reply:
I think so too. It will look better. I will update it.